### DISCUSSION TITLE
Proofs found no rewards

### DISCUSSION THREAD

**Discussion by sp45** [Chia-Network/chia-blockchain#7567] [Support]
I have proof pound and have received zero rewards
Whats the fix, as I have at least 8 of these in the past hour.

![image](https://user-images.githubusercontent.com/48410719/125952905-26e18584-048e-49ef-9c2a-ba1e8a05f70d.png)

**Comment by alexwwstx** [comment]
are you in a pool?

**Reply by sp45** [reply]
I'm in  pool, but the plots that are being found are none pool plots

**Reply by sp45** [reply]
In process monitor you can see disk activity, the plots are on a drive that is not on a pool

**Reply by rmica** [reply]
Did you search for strings like "Starting to make block"  in the logs? It should happen at the same second of the found proof. I don't know the exact meaning of  "Proofs Found" in the GUI, but I guess it's the harvester's information and not the farmer's one. So you should confirm that your farmer/node has done the actual job of successfully signing the block.

**Comment by alexwwstx** [comment]
How do you tell it’s from OG, not pool?

**Reply by rmica** [reply]
I had the same problem and opted for splitting NFT/OG plots on separate harvesters, so that I can immediately know even with one farmer the type of found proofs.
